### ISSUE TITLE
Mushroom farm worksite blocks do not work

### ISSUE THREAD

**Issue by Izidore** [P3pp3rF1y/AncientWarfare2#240]
- Mushroom worksites do not harvest or plant mushrooms/netherwart.
- Torque power seems to connect but no 'work' is accomplished.
- NPC Farmers do not interact with worksite.
- All other worksites seem to function properly.

**Comment by GotoLink** [comment]
Mushroom farm follow same block placement rules as a player.
Nether wart goes on soul sand, mushrooms in low light.

**Comment by Izidore** [comment]
I do not understand your first comment about player block placement.
I will try to replicate the issue but I did some extensive testing....
I understand the requirements to grow mushrooms.
I planted the farmed area full of mushrooms and they would not 'harvest' either if planted manually or grown 'naturally'.
I could not get the farm block to plant mushrooms under any circumstances.
The farm block appears to accept 'power' through a shaft but no work (ie; plant or harvest) is ever accomplished.  NPC farmers will not interact with the block.
